helm/charts/text-generation-inference/values.yaml

```diff
@@ -0,0 +1,62 @@
+---
+replicaCount: 1
+
+image:
+  repository: ghcr.io/huggingface/text-generation-inference
+  tag: "1.1.0"
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+resources:
+  limits:
+    nvidia.com/gpu: "1"
+  requests:
+    nvidia.com/gpu: "1"
+
+args:
+  []
+  # See: https://huggingface.co/docs/text-generation-inference/basic_tutorials/launcher
+  # - "--model-id"
```

Comment:
we could alternatively provide a simpler way to define the model to use, example:
```
model:
  id: bigcode/starcoderbase-7b
  revision: 4ab631381edb607557cbb04b6e9a225bad16807c
```
This way  user do not need to know the exact args to provide.

Reply:
Intuitively I would have done it like you're proposing, but args doesn't change much and is closer to the real thing, no strong opinion on my side.